src/components/MobileHeaderBar.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <q-toolbar class="bg-primary text-white">
+    <q-btn flat round dense>
+      <q-icon name="menu" />
+      <q-menu transition-show="none" transition-hide="none">
+        <q-list dense>
+          <menu-item
+            v-for="(item, index) in menudata"
+            :key="item.label"
+            :menudata="item"
+            v-model:selected="subMenuOpenFlags[index]"
+          />
+        </q-list>
+      </q-menu>
+    </q-btn>
+    <q-space />
+    <q-btn
+      v-for="button in headerButtons"
+      :key="button.icon"
+      :disable="button.disable"
+      flat
+      round
+      dense
+      @click="button.onClick"
+      :icon="button.icon"
+    />
+  </q-toolbar>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import { MenuItemData } from "./MenuBar.vue";
+import MenuItem from "./MenuItem.vue";
+import { useStore } from "@/store";
+const store = useStore();
+
+const uiLocked = computed(() => store.getters.UI_LOCKED);
+const canUndo = computed(() => store.getters.CAN_UNDO);
+const canRedo = computed(() => store.getters.CAN_REDO);
+
```

Comment:
忘れないようにwarn書いとくのどうでしょ

```suggestion

store.dispatch("LOG_WARN", "PC版のヘッダーと統合するか要検討");

```

Reply:
うーん、まぁTODOコメントあたりで十分だと思います。